src/storage.rs

```diff
@@ -754,6 +877,29 @@ fn process_imported_audits(
                     }
                 }
             }
+            for (package, existing_audits) in &existing_audits_file.wildcard_audits {
+                let new_audits = audits_file
+                    .wildcard_audits
+                    .get_mut(package)
+                    .map(|v| &mut v[..])
+                    .unwrap_or(&mut []);
+                for existing_audit in existing_audits {
+                    for new_audit in &mut *new_audits {
+                        // Ignore `who` and `notes` for comparison, as they
+                        // are not relevant semantically and might have been
+                        // updated uneventfully.
+                        if new_audit.is_fresh_import
+                            && new_audit.user_id == existing_audit.user_id
+                            && new_audit.start == existing_audit.start
+                            && new_audit.end == existing_audit.end
+                            && new_audit.criteria == existing_audit.criteria
```

Comment:
I'd like to see this as a method on `WildcardEntry` so that if that is changed in the future, the potential to miss changing the corresponding equality is less.

Reply:
Sure, moving it to a `same_audit_as` method.